classes/class-swsales-reports.php

```diff
@@ -328,6 +328,10 @@ public static function enqueue_tracking_js() {
 			return;
 		}
 
+		if ( null === $active_sitewide_sale->is_running() ) {
```

Comment:
```suggestion
		if ( ! $active_sitewide_sale->is_running() ) {
```
`$active_sitewide_sale->is_running() ` should return a boolean